_includes/components/two_column_image_bullet_list.html

```diff
@@ -0,0 +1,43 @@
+{% comment %}
+includes
+- column1-image
+- column1-header
+
+- class
+- col1
+- bannertext
+- bannercol1
+- bannercol2
+- col2
+{% endcomment %}
+<div class="grid-row grid-gap measure-3">
+
+  <div class="tablet:grid-col">
+    <img alt="" width="30%" style="margin-left: 35%" src="{{ site.baseurl }}{{ include.column1-image }}" />
+    <div class="{{ include.column1-class }}">
+      <div class="text-bold">{{ include.column1-header }}</div>
+      <ul class="usa-list">
+        {% assign bullets = include.column1-bullets | split: ", " %}
+        {% for bullet in bullets %}
+        <li class="font-body-xs">{{ bullet }}</li>
+        {% endfor %}
+      </ul>
+      <p class="font-body-xs">{{ include.column1-footer }}</p>
+    </div>
+  </div>
+
+  <div class="tablet:grid-col">
+    <img alt="" width="30%" style="margin-left: 35%" src="{{ site.baseurl }}{{ include.column2-image }}" />
+    <div class="{{ include.column2-class }}">
+      <div class="text-bold">{{ include.column2-header }}</div>
+      <ul class="usa-list">
+        {% assign bullets = include.column2-bullets | split: ", " %}
```

Comment:
I would consider using a character like `|` to split, if we are unable to pass an array, because `,` seems pretty likely to end up as part of a normal sentence

Reply:
Point.

Reply:
Changed, and applied the new partials to the Spanish and French versions of the page.